src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -334,6 +352,7 @@ public GerritTrigger(List<GerritProject> gerritProjects, SkipVote skipVote, Inte
         this.gerritTriggerTimerTask = null;
         this.triggerInformationAction = new GerritTriggerInformationAction();
         this.notificationLevel = notificationLevel;
+        this.silentLevel = silentLevel;
```

Comment:
This constructor is deprecated and should not be added to. use `@DataboudSetter`s instead.